### PR TITLE
Customer Examples Library

### PR DESCRIPTION
**Pull request by EmilyNecciai**
# Pull Request/Issue Resolution

**Description of Change:**
> I'm changing..... (could be a link, a new image, a new section, etc.)...

**Reason for Change:**
> I'm making this change because.....

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Feature Release Date:__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

## PR Checklist
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @EmilyNecciai as a reviewer when the your work is _done and ready to be reviewed for merge_.
- [ ] Consult the [Docs Text Formatting Guide](https://github.com/Appboy/success/wiki/Docs-Text-Formatting-Guide).
- [ ] Consult the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/success/wiki/Writing-Style-Guide-&-Best-Practices).
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

## ATTENTION: REVIEWERS OF THIS PR
- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
---
---

### CODE REVIEW

File: _docs/_help/best_practices/client_integration_gallery.md [hunk 1]
```diff
@@ -1,182 +1,44 @@
 ---
-nav_title: Client Integration Gallery
-page_order: 9
+page_order: 20
+nav_title: Customer Examples Library
+layout: glossary_page
+glossary_top_header: "Customer Examples Library"
+glossary_top_text: "See some shining examples of real messages using our Campaigns and Canvas tools!"
```
Comment (EmilyNecciai) [inline _docs/_help/best_practices/client_integration_gallery.md]: @JSB1861 
Feel free to revise

File: _docs/_help/best_practices/client_integration_gallery.md [hunk 2]
```diff
@@ -1,182 +1,44 @@
...
+
+glossary_tag_name: "Industry"
+glossary_filter_text: "Select tags below to narrow the library."
+
+glossary_tags:
```
Comment (EmilyNecciai) [inline _docs/_help/best_practices/client_integration_gallery.md]: List of industries

File: _docs/_help/best_practices/client_integration_gallery.md [hunk 3]
```diff
@@ -1,182 +1,44 @@
...
+  - name: Media & Entertainment
+  - name: Retail & eCommerce
+  - name: Financial Services & Insurance
+  - name: Delivery & Transportation
+  - name: Miscellaneous & Other
+
+
+glossaries:
+  - name: "Re-Engage Lapsed Users with a Personalized Push to a Wide Audience"
```
Comment (EmilyNecciai) [inline _docs/_help/best_practices/client_integration_gallery.md]: Title - click-baitish

File: _docs/_help/best_practices/client_integration_gallery.md [hunk 4]
```diff
@@ -1,182 +1,44 @@
...
+    image: /docs/assets/img/1800-flowers_alerts_push.jpg
```
Comment (EmilyNecciai) [inline _docs/_help/best_practices/client_integration_gallery.md]: image: /docs/assets/img/[imagename.png]

File: _docs/_help/best_practices/client_integration_gallery.md [hunk 5]
```diff
@@ -1,182 +1,44 @@
...
+    description: "<a href='https://www.1800flowers.com/'>1800 Flowers</a> <br> <br> This push message reminds your user that, after a few weeks of no purchases or between holidays, there are still flowers just waiting to be bought! Who cares who they're for - there is no need for a special occasion! Treat your self. <br> As you can see, this message uses personalization and emojis to really connect with customers."
```
Comment (EmilyNecciai) [inline _docs/_help/best_practices/client_integration_gallery.md]: [LInked Customer Name to customer site]
2 line breaks
Description.

File: _docs/_help/best_practices/client_integration_gallery.md [hunk 6]
```diff
@@ -1,182 +1,43 @@
...
+glossary_tags:
+  - name: Media & Entertainment
+  - name: Retail & eCommerce
+  - name: Financial Services & Insurance
+  - name: Delivery & Transportation
+  - name: Miscellaneous & Other
+
+
+glossaries:
+  - name: "Use an Engaging Email to Share New Product Features with Your Users!"
```
Comment (JSB1861) [inline _docs/_help/best_practices/client_integration_gallery.md]: @EmilyNecciai I've entered the name, description, image, and tags in for one item. I'm not sure the image is correct. I wanted you to be able to review this before I entered anything else in Github. I'm also emailing you with some non-Github questions.

**Comment by zzhaobraze** [comment]
@cla-bot check

**Comment by EmilyNecciai** [comment]
@JSB1861 Something is still up with this PR. I'll close this one and tag you on a new one. Sorry about that!

**Pull request closed without merging**